src/observers/models/transformers.py

```diff
@@ -0,0 +1,48 @@
+import uuid
+from typing import Any, Dict, List, Optional, Union
+
+import transformers
+from typing_extensions import Self
+
+from observers.models.base import ChatCompletionObserver, ChatCompletionRecord
+from observers.stores.datasets import DatasetsStore
+from observers.stores.duckdb import DuckDBStore
+
+
+class TransformersRecord(ChatCompletionRecord):
```

Comment:
I will try to see if I can add some Argilla configs here in a separate PR :)

Reply:
Also, I cill create a separate PR to allow for adding the sampler i.e. logging only x percentage.